guardian/manage-frontend
# Upgrade journey: Create suggested amounts getter

## What does this change?

Adds a lookup object for suggested amounts for the current amount a reader is supporting with before entering the upgrade support journey. Each currency/monthly or annual combination is a function that calculates the 3 amounts to suggest.

[Source spreadsheet](https://docs.google.com/spreadsheets/d/1lxnOJhSPrddZmdjNe85sbTCtprqQlSy6_QcwY_n7AFk/edit#gid=0)



## How to test

Go to `/upgrade-support` and see the 3 amounts that come from a spreadsheet.

How do we test there isn't a mistake in this implementation... (not sure)



## How can we measure success?



## Have we considered potential risks?

Creating this was a lot of manual data entry! Not good for maintainability in future



## Images

<img width="574" alt="image" src="https://github.com/guardian/manage-frontend/assets/114918544/e819dbf3-049e-4885-9139-be6b928de2de">



## Accessibility
